docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04/index.md

```diff
@@ -0,0 +1,142 @@
+ - '[Composer Documentation](https://getcomposer.org/doc/)'
+---
+
+[Drush](https://www.drush.org/) is a command line tool for creating, administrating, and modifying Drupal websites. Command line tools, like Drush, add functionality through additional command packages. Once installed, Drush is as easy to use as any of the basic Linux commands. The name comes from combining the words Drupal and shell. Drush is designed only for Drupal and cannot be used with other content management systems.
+
+Both new and experienced Drupal users can benefit from learning Drush. Users that have worked with a command line interface before have an advantage, but Drush is an excellent application for beginners, too.
+
+## Before You Begin
+
+Before installing Drush, ensure that the following prerequisites have been met:
+
+1.  [Create a new Linode](/docs/getting-started/#create-a-linode) by following our [Getting Started](/docs/getting-started/) guide. Complete the steps for [setting your Linode's hostname](/docs/getting-started/#set-the-hostname) and [timezone](/docs/getting-started/#set-the-timezone).
+1. Follow our [Securing Your Server](/docs/security/securing-your-server) guide to [create a standard user account](/docs/security/securing-your-server/#add-a-limited-user-account), [harden SSH access](/docs/security/securing-your-server/#harden-ssh-access), [remove unnecessary network services](/docs/security/securing-your-server/#remove-unused-network-facing-services) and [create firewall rules](/docs/security/securing-your-server/#configure-a-firewall) for your Linode.
+1.  Install and configure a [LAMP stack on Ubuntu 18.04](/docs/web-servers/lamp/how-to-install-a-lamp-stack-on-ubuntu-18-04/)
+1.  Make sure that your system is up to date, using:
+
+        sudo apt-get update && sudo apt-get upgrade
+
+    {{< note >}}
+This guide is written for a non-root user. Commands that require elevated privileges are prefixed with ``sudo``. If you're not familiar with the ``sudo`` command, you can check our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+    {{< /note >}}
+
+## Install Git & Composer
+
+The developers of Drush recommend installing Drush using [Composer](https://getcomposer.org/doc/00-intro.md), a PHP dependency manager. Since the Drush project is hosted on [GitHub](https://github.com/) and controlled with [Git](/docs/development/version-control/how-to-configure-git/), you will also need to install Git. In this section, you will install both dependencies.
+
+1.  Install Git:
+
+        sudo apt-get install git
+
+1.  Install Composer:
+
+        curl -sS https://getcomposer.org/installer | php
+
+1.  Move the composer.phar file to `/usr/local/bin/`, so that it can be accessed from any directory:
+
+        sudo mv composer.phar /usr/local/bin/composer
+
+1. Composer has a few PHP dependencies it needs in order to run. Install them on your Ubuntu system:
+
+        sudo apt-get install php-xml, zip
```

Comment:
Is the comma here going to result in an error?

Reply:
no